spinnman/processes/get_machine_process.py

```diff
@@ -171,52 +185,259 @@ def get_machine_details(self, boot_x, boot_y, width, height,
                 logger.warning(
                     "Chip {}, {} was expected but didn't reply", x, y)
 
-        return self.create_machine(
-            width, height, repair_machine, ignore_bad_ethernets)
+        self._preprocess_ignore_chips(machine)
+        self._process_ignore_links(machine)
+        self._preprocess_ignore_cores(machine)
 
-    def create_machine(
-            self, width, height, repair_machine, ignore_bad_ethernets):
-        bad_ethernets = []
-        machine = machine_from_size(width, height)
+        return self._fill_machine(
+            machine, repair_machine, ignore_bad_ethernets)
+
+    def _fill_machine(
+            self, machine, repair_machine, ignore_bad_ethernets):
         for chip_info in sorted(
                 self._chip_info.values(), key=lambda chip: (chip.x, chip.y)):
-            if (chip_info.x, chip_info.y) not in self._ignore_chips:
-                if (chip_info.ethernet_ip_address is not None and
+            if (chip_info.x, chip_info.y) in self._ignore_chips:
+                logger.warning(
```

Comment:
are we sure we want to warn everytime a chip is ignored?????

Reply:
This bit of code is dead as _preprocess_ignore_chips removed the chip_info.

_preprocess_ignore_chips does have an log info which I am leaving is.
Ony people who really use ignores are testes and they want info.

Reply:
same argument as one below

Reply:
stop arguing about code that has been removed.